File: ejercicio 1/listaSupermercado.js
let compras = ["Manzanas"];

// 1. Añadir "Leche" y "Pan" al FINAL
compras.push("Leche", "Pan");

// 2. Añadir "Huevos" al INICIO
compras.unshift("Huevos");

// 3. Eliminar "Manzanas" (segundo elemento)
compras.splice(1, 1);

console.log(compras);  // ["Huevos", "Leche", "Pan"]

File: ejercicio 1/listaSupermercado.java
import java.util.*;

public class ListaSupermercado {
    public static void main(String[] args) {
        ArrayList<String> compras = new ArrayList<>(Arrays.asList("Manzanas"));
        
        // 1. Añadir "Leche" y "Pan" al final
        compras.add("Leche");
        compras.add("Pan");
        
        // 2. Añadir "Huevos" al inicio
        compras.add(0, "Huevos");
        
        // 3. Eliminar "Manzanas" (segundo elemento)
        compras.remove(1);
        
        System.out.println(compras);
    }
}
